lib/_internal/file_picker_web.dart

```diff
@@ -185,6 +186,45 @@ class FilePickerWeb extends FilePicker {
     return files == null ? null : FilePickerResult(files);
   }
 
+  @override
+  Future<String?> saveFile({
+    String? dialogTitle,
+    String? fileName,
+    String? initialDirectory,
+    FileType type = FileType.any,
+    List<String>? allowedExtensions,
+    Uint8List? bytes,
+    bool lockParentWindow = false,
+  }) async {
+    if (bytes == null) {
+      throw ArgumentError(
```

Comment:
The indent is a bit off here.

```suggestion
      throw ArgumentError(
        'The bytes are required when saving a file on the web.',
      );
```